src/main/java/uk/gov/hmcts/reform/unspec/service/flowstate/StateFlowEngine.java

```diff
@@ -70,81 +79,83 @@
+                .transitionTo(COUNTER_CLAIM).onlyIf(counterClaimAfterAcknowledge)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
+                .transitionTo(CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE).onlyIf(caseDismissedAfterClaimAcknowledged)
+            .state(NOTIFICATION_ACKNOWLEDGED_TIME_EXTENSION)
+                .transitionTo(FULL_DEFENCE).onlyIf(fullDefence)
+                .transitionTo(FULL_ADMISSION).onlyIf(fullAdmission)
+                .transitionTo(PART_ADMISSION).onlyIf(partAdmission)
+                .transitionTo(COUNTER_CLAIM).onlyIf(counterClaim)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
+                .transitionTo(CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE).onlyIf(caseDismissedAfterClaimAcknowledged)
+            .state(FULL_DEFENCE)
                 .transitionTo(FULL_DEFENCE_PROCEED).onlyIf(fullDefenceProceed)
                 .transitionTo(FULL_DEFENCE_NOT_PROCEED).onlyIf(fullDefenceNotProceed)
-                .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
-                .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
-                .transitionTo(CASE_PROCEEDS_IN_CASEMAN).onlyIf(caseProceedsInCaseman)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
                 .transitionTo(TAKEN_OFFLINE_PAST_APPLICANT_RESPONSE_DEADLINE).onlyIf(applicantOutOfTime)
-            .state(RESPONDENT_FULL_ADMISSION)
-                .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
-                .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
-                .transitionTo(CASE_PROCEEDS_IN_CASEMAN).onlyIf(caseProceedsInCaseman)
-                .transitionTo(PROCEEDS_OFFLINE_ADMIT_OR_COUNTER_CLAIM).onlyIf(claimTakenOffline)
-            .state(RESPONDENT_PART_ADMISSION)
-                .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
-                .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
-                .transitionTo(CASE_PROCEEDS_IN_CASEMAN).onlyIf(caseProceedsInCaseman)
-                .transitionTo(PROCEEDS_OFFLINE_ADMIT_OR_COUNTER_CLAIM).onlyIf(claimTakenOffline)
-            .state(RESPONDENT_COUNTER_CLAIM)
-                .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
-                .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
-                .transitionTo(CASE_PROCEEDS_IN_CASEMAN).onlyIf(caseProceedsInCaseman)
-                .transitionTo(PROCEEDS_OFFLINE_ADMIT_OR_COUNTER_CLAIM).onlyIf(claimTakenOffline)
+            .state(FULL_ADMISSION)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
+            .state(PART_ADMISSION)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
+            .state(COUNTER_CLAIM)
+                .transitionTo(TAKEN_OFFLINE_BY_STAFF).onlyIf(takenOfflineByStaff)
```

Comment:
I don't think this transition is required, please double check with @ckalista

Reply:
yes, it's not required as the flow go to offline on it's own